### ISSUE TITLE
Regarding exam

### ISSUE THREAD

**Issue by Harrypotter098** [DerekBabb/CyberSecurity#23]
I am in 12th now can I get into cybersecurity without jee exam

**Comment by bikram8** [comment]
> I am in 12th now can I get into cybersecurity without jee exam

Yes sure , there is no any specific timing to enter in this field you can explore the things but graduation is also important with minimum 60 - 70 percentage.
